spec.md

```diff
@@ -586,18 +589,68 @@ schema being used.
           "40 concurrent connections"
         ]
       }
+    }, {
+      "name": "fake-deprecated-plan",
+      "id": "009aba-XXXX-XXXX-XXXX-dace631cd648",
+      "description": "This is deprecated.",
+      "free": false,
+      "deprecation": {
+        "description": "Don't use this anymore.",
+        "since": "2018-04-20T17:00:00+00:00",
+        "eol": "2020-02-28T08:00:00+00:00",
+        "alternatives": [{
+            "description": "Use this plan instead.",
+            "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648"
+          }
+        ]
+      }
     }]
   }]
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| description | string | A short description of the deprecation. |
+| since | date string | Date and time when the Service Broker deprecated the service or plan. If specified, it is expected to be an ISO 8601 formatted date and time, with time zone. |
```

Comment:
s/it is expected to be/it MUST be/

Reply:
Its odd that none of these fields are required. Was that intentional? Seems like at least one of them should be - if not both 'since' and 'eol'. Even if the 'eol' is 2100, at least that gives people a sense of how quickly they need to move.

Reply:
If the service/plan is just deprecated, then `deprecation` should be an empty object. The existence of `deprecation` on the plan or class means it is deprecated. It is up to the broker to give more hints.